test/packages/npm-test-shrinkwrap/test.js

```diff
@@ -1,4 +1,4 @@
-var path = require("path")
```

Comment:
test/tap/legacy-shrinkwrap.js below had it's behavior changed

I will delete this file after confirming it is dead code.